api/v1beta2/moduletemplate_types.go

```diff
@@ -189,18 +189,38 @@ func (m *ModuleTemplate) IsInternal() bool {
 
 var ErrInvalidVersion = errors.New("can't find valid semantic version")
 
-func (m *ModuleTemplate) GetVersion() (*semver.Version, error) {
+// GetVersionLegacy() returns the version of the ModuleTemplate from the annotation on the object.
+// Remove once shared.ModuleVersionAnnotation is removed
+func (m *ModuleTemplate) GetVersionLegacy() (string, error) {
```

Comment:
not need to make it public, it's a private method now.

Reply:
done.